src/fu-util.c

```diff
@@ -2545,12 +2545,20 @@ fu_util_check_daemon_version (FuUtilPrivate *priv, GError **error)
 	const gchar *daemon = fwupd_client_get_daemon_version (priv->client);
 
 	if (g_strcmp0 (daemon, SOURCE_VERSION) != 0) {
-		g_set_error (error,
-			     FWUPD_ERROR,
-			     FWUPD_ERROR_NOT_SUPPORTED,
-			     /* TRANSLATORS: error message */
-			     _("Unsupported daemon version %s, client version is %s"),
-			     daemon, SOURCE_VERSION);
+		if (daemon == NULL) {
```

Comment:
i think this should go before the `if (g_strcmp0 (daemon, SOURCE_VERSION`

Reply:
fixed